sdk/src/manifest_store.rs

```diff
@@ -89,22 +89,41 @@ impl ManifestStore {
     // writes a resource identified uri to the given stream
     pub fn get_resource(&self, uri: &str, stream: impl Write + Read + Seek + Send) -> Result<u64> {
         // get the manifest referenced by the uri, or the active one if None
-        let manifest = match manifest_label_from_uri(uri) {
-            Some(label) => self.get(&label),
-            None => self.get_active(),
+        // add logic to search for local or absolute uri identifiers
+        let (manifest, label) = match manifest_label_from_uri(uri) {
+            Some(label) => (self.get(&label), label),
+            None => (
+                self.get_active(),
+                self.active_label().unwrap_or_default().to_string(),
+            ),
         };
+        let relative_uri = to_relative_uri(uri);
```

Comment:
This seems to have become a common pattern.  Ensure it is relative then make absolute.

let absolute_uri = if uri.isrelative() { to_absolute_uri(&label, uri) } else { uri.to_owned());

We could move this logic to inside of the to_absolute_uri.